client/src/components/EditReviewForm.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorList"
+import translateServerErrors from "../services/translateServerErrors"
+
+const EditReviewForm = (props) => {
+  const [editReview, setEditReview] = useState({
+    body: "",
+    score: "",
+    userId: "",
+    reviewId: "",
+    showId: ""
+  })
+
+  const [errors, setErrors] = useState([])
+
+  const userId = props.userId
+
+  const handleInputChange = (event) => {
+    setEditReview({
+      ...editReview,
+      [event.currentTarget.name]: event.currentTarget.value,
+      userId: userId,
+      reviewId: props.reviewId,
+      showId: props.showId
+    })
+  }
+
+  const showId = props.showId
```

Comment:
good. we want to do this over storing showId in state. 

you can make this DRYer by destructoring your props as you define the component as well (optional)